src/store/PriceLineStore.ts

```diff
@@ -2,7 +2,7 @@ import EventEmitter from 'event-emitter-es6';
 import { action, computed, observable, when, makeObservable, reaction, IReactionDisposer } from 'mobx';
 import Context from 'src/components/ui/Context';
 import MainStore from '.';
-import { ARROW_HEIGHT, DIRECTIONS, makeElementDraggable } from '../utils';
+import { ARROW_HEIGHT, DIRECTIONS, lerp, makeElementDraggable } from '../utils';
```

Comment:
whats lerp lol

Reply:
could this be changed to something more descriptive

Reply:
lerp is linear interpolation 😄 
https://docs.unity3d.com/ScriptReference/Vector3.Lerp.html